site/src/layouts/mdx.tsx

```diff
@@ -190,9 +190,12 @@ const MdxLayout: React.FC<MdxLayoutProps> = (props) => {
     ) {
       /* eslint-disable no-param-reassign */
       mdxNode.isBookmark = false;
-      mdxNode.href = `/${props.prefix}${
-        mdxNode.parent.name === 'index' ? '' : `/${mdxNode.parent.name}`
-      }`;
+      if (mdxNode.parent.name === 'index') mdxNode.href = `/${props.prefix}`;
+      else if (typeof mdxNode.parent.relativeDirectory === 'undefined')
+        mdxNode.href = `/${props.prefix}/${mdxNode.parent.name}`;
+      else {
```

Comment:
Let's always use curly braces for `if` and other blocks.
```suggestion
      if (mdxNode.parent.name === 'index') {
        mdxNode.href = `/${props.prefix}`;
      } else if (typeof mdxNode.parent.relativeDirectory === 'undefined') {
        mdxNode.href = `/${props.prefix}/${mdxNode.parent.name}`;
      } else {
```